internal/jobservice/eventstojobs/event_job_response.go

```diff
@@ -0,0 +1,56 @@
+package eventstojobs
+
+import (
+	"fmt"
+
+	"github.com/G-Research/armada/pkg/api"
+	js "github.com/G-Research/armada/pkg/api/jobservice"
+)
+
+func EventsToJobResponse(message api.EventMessage) (*js.JobServiceResponse, error) {
+	switch message.Events.(type) {
+	case *api.EventMessage_Submitted:
+		return &js.JobServiceResponse{State: js.JobServiceResponse_SUBMITTED}, nil
+	case *api.EventMessage_DuplicateFound:
+		return &js.JobServiceResponse{State: js.JobServiceResponse_DUPLICATE_FOUND}, nil
+	case *api.EventMessage_Running:
+		return &js.JobServiceResponse{State: js.JobServiceResponse_RUNNING}, nil
+	case *api.EventMessage_Failed:
+		return &js.JobServiceResponse{State: js.JobServiceResponse_FAILED, Error: message.GetFailed().Reason}, nil
+	case *api.EventMessage_Succeeded:
+		return &js.JobServiceResponse{State: js.JobServiceResponse_SUCCEEDED}, nil
+	case *api.EventMessage_Cancelled:
+		return &js.JobServiceResponse{State: js.JobServiceResponse_CANCELLED}, nil
+	}
+
+	return nil, fmt.Errorf("unknown event type")
```

Comment:
why are we returing errros here? Isn't it expected that there will be events which we want to ignore?  I'd suggest just silently ignoring any events we don;'t care about.

Reply:
To be more precise- I'd get rid of `IsEventAJobResponse` and just call this method regardless which will return nil if it's not an event we care about.

Reply:
Good idea!  Done.